packages/credentials/src/issuer.ts

```diff
@@ -134,3 +144,64 @@ export async function issue({
       )
   }
 }
+
+/**
+* Revokes a Kilt credential on the blockchain, making it invalid.
+* 
+* @param attester The DID URL of the attester revoking the credential
+* @param submitterAccount The account that will submit and pay for the transaction
+* @param signCallback Callback function to sign the DID authorization 
+* @param credential The Verifiable Credential to be revoked
+* @returns An object containing:
+* - success: Boolean indicating if revocation was successful
+* - error?: Array of error messages if revocation failed
+* - info: Object containing blockchain transaction details:
+*   - blockNumber?: Block number where transaction was included
+*   - blockHash?: Hash of the block
+*   - transactionHash?: Hash of the transaction
+*/
+export async function revoke(
+  attester: DidUrl,
+  submitterAccount: KeyringPair,
+  signCallback: any,
+  credential: VerifiableCredential,
```

Comment:
I'd like to only see two parameters here:
```suggestion
  credential: VerifiableCredential,
  issuer: IssuerOptions,
```
This is enough for issuance, and will also be enough for revocation.

Reply:
Fixed at: https://github.com/KILTprotocol/sdk-js/pull/918/commits/ccac9b90e1ceac14a1c26d443d3fcef994745055